### PR TITLE
fix: include defaults in subscription update

### PR DESCRIPTION
**Pull request by dnikolay-ebc**
set exclude_defaults=False in update_subscription and wrote a unit test

### CODE REVIEW

File: tests/clients/test_ngsi_v2_cb.py [hunk 1]
```diff
@@ -338,6 +340,50 @@ def test_subscriptions(self):
...
+                },
+                "provider": {
+                    "http": {
+                        "url": "http://localhost:1234",
+                    },
+                    "supportedForwardingMode": "all"
+                }
+            }
+        reg = Registration(**reg_example)
+        reg_id = client.post_registration(registration=reg)
```
Comment (RedingJeff) [inline tests/clients/test_ngsi_v2_cb.py:369]: Registration is not imported. And I get a parsing error for http

@tstorek are the reworked models already merged? I am not sure

Comment (tstorek) [reply tests/clients/test_ngsi_v2_cb.py]: @dnikolay-ebc I added the new models to the development. Please try to merge the development into your branch

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+#### v0.1.9
+- include default values in subscription update ([#39](https://github.com/RWTH-EBC/FiLiP/issues/39))
```
Comment (tstorek) [inline CHANGELOG.md:2]: since we did not yet release version 0.1.8 it should be part of 0.1.8

File: setup.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
 setuptools.setup(
     name='filip',
-    version='0.1.8',
+    version='0.1.9',
```
Comment (tstorek) [inline setup.py:24]: this will be done with the next release

**Review by tstorek — CHANGES_REQUESTED** [review]
please do not update the version yet. in case you need to update the version please do not forget the version in the module and the docs

**Review by tstorek — APPROVED** [review]

**Pull request merged**